js_modules/dagster-ui/packages/ui-core/src/runs/RunTimelineRowIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import {Icon} from '@dagster-io/ui-components';
+
+import {RowObjectType} from './RunTimelineTypes';
+import {assertUnreachable} from '../app/Util';
+
+export const RunTimelineRowIcon = ({type}: {type: RowObjectType}) => {
+  switch (type) {
+    case 'asset':
+      return <Icon name="asset" />;
+    case 'job':
+      return <Icon name="job" />;
+    case 'manual':
+      return <Icon name="account_circle" />;
```

Comment:
@braunjj Thoughts on this as the "manually launched" icon?

Reply:
I think its fine